src/main/java/org/launchcode/mentalhealthcareaccess/models/data/dto/RegisterFormDTO.java

```diff
@@ -0,0 +1,64 @@
+package org.launchcode.mentalhealthcareaccess.models.data.dto;
+
+import javax.validation.constraints.NotBlank;
+
+public class RegisterFormDTO extends LoginFormDTO {
+    @NotBlank
+    private String verifyPassword;
+    @NotBlank
+    private String firstName;
+    private String lastName;
+    private String companyName;
+    private String displayName;
```

Comment:
I don't think this field is needed within the DTO, only on the model class. Let me know if you think otherwise, but I'm not seeing it used here, or a potential use case in the future.

Reply:
It's being used in `ProviderController`

```
Provider newProvider = new Provider(
                registerFormDTO.getDisplayName(),
                registerFormDTO.getCompanyName(),
                registerFormDTO.getFirstName(),
                registerFormDTO.getLastName(),
                registerFormDTO.getEmail(),
                registerFormDTO.getPhoneNumber(),
                registerFormDTO.getPassword());
```

I have the methods for setting displayName in `Provider` and `registerFormDTO`. I can't seem to get them to work when they are just in `Provider`. I'll keep working on checking what's wrong here.